src/ArbCall/ArbFPWrapFunction.jl

```diff
@@ -171,6 +177,22 @@ function jlcode(af::ArbFPWrapFunction, jl_fname = jlfname(af))
     return func
 end
 
+"""
+    @arbfpwrapcall_str str
+
+Parse a string as an [`Arblib.ArbCall.ArbFPWrapFunction`](@ref),
+generate the code for a corresponding method with
+[`Arblib.ArbCall.jlcode`](@ref) and evaluate the code.
+
+For example
+```
+arbfpwrapcall"int arb_fpwrap_double_exp(double * res, double x, int flags)"
+```
+defines the method
+```
+fpwrap_exp(x::Union{Float16, Float32, Float64}; error_on_failure::Bool = Arblib.ArbCall.fpwrap_error_on_failure_default(), correct_rounding::Bool = false, work_limit::Integer = 8)
```

Comment:
just whitespace to distinguish args from kwargs:
```suggestion
fpwrap_exp(x::Union{Float16, Float32, Float64};
    error_on_failure::Bool = Arblib.ArbCall.fpwrap_error_on_failure_default(),
    correct_rounding::Bool = false, work_limit::Integer = 8
)
```

Reply:
Fixed!